src/proc.c

```diff
@@ -191,6 +191,30 @@ int print_proc_maps(int pid)
     return system(cmd);
 }
 
+int print_proc_comm(int pid)
+{
+    FILE *f;
+    char buf[32] = "";
```

Comment:
I believe an earlier implementation required it.  It's probably not needed now, however, I feel safer making sure a string is properly terminated.